Jenkinsfile

```diff
@@ -1,52 +1,47 @@
 pipeline {
-  agent any
+  agent {
+    docker {
+      image 'px4io/px4-docs:2018-06-14'
+    }
+  }
   stages {
 
     stage('Build') {
       environment {
         HOME = "${WORKSPACE}"
       }
 
-      agent {
-        docker {
-          image 'px4io/px4-docs:2018-06-14'
-        }
-      }
-
       steps {
-        sh 'export'
-        sh 'gitbook install'
-        sh 'gitbook build'
-        stash includes: '_book/', name: 'gitbook'
+        sh('export')
+        checkout(scm)
+        sh('gitbook install')
+        sh('gitbook build')
+        stash(includes: '_book/', name: 'gitbook')
         // publish html
-        publishHTML target: [
+        publishHTML(target: [
           reportTitles: 'PX4 Dev Guide',
           allowMissing: false,
           alwaysLinkToLastBuild: true,
           keepAll: true,
           reportDir: '_book',
           reportFiles: '*',
           reportName: 'PX4 Dev Guide'
-        ]
+        ])
       }
 
     } // Build
 
     stage('Deploy') {
       environment {
+        GIT_AUTHOR_EMAIL = "bot@pixhawk.org"
```

Comment:
This is needed because there is not checkout stage, to prevent git errors not knowing who the author is.